### ISSUE TITLE
cannot access fields on user-defined functions

### ISSUE THREAD

**Issue by cristobaltapia** [cetz-package/cetz-plot#65]
Hi, I was trying the examples in the manual, but could not compile any of them. I get the following error:

```
error: cannot access fields on user-defined functions
   ┌─ test.typ:12:11
   │
12 │       plot.add(domain: (0, calc.pi), calc.sin)
   │            ^^^
```

I am using the following code:

```typst
#import "@preview/cetz:0.3.0"
#import "@preview/cetz-plot:0.1.0": plot, chart

#cetz.canvas({
  import cetz.draw: *
  import cetz.plot: *

  plot.plot(
    size: (5, 5),
    axis-style: "scientific",
    {
      plot.add(domain: (0, calc.pi), calc.sin)
    },
  )

})
```

What might be the problem? It seems I am the only one having this issue.

**Comment by johannes-wolf** [comment]
Remove the `import cetz.plot: *` line. I guess you accidentally overwrote the `plot` module by importing it from `cetz` (which contains a stub-function).

**Comment by cristobaltapia** [comment]
Oh, thanks. That was it. In that case, there is a bug in the pdf manual (see page 3).

**Comment by johannes-wolf** [comment]
Fixed with #69.
